### PR TITLE
Updates and fixes for custom DC UI/deployment

### PR DESCRIPTION
**Pull request by shifucun**
deploy_gke.sh is updated with named args, instead of positional args now. All references should be updated.

Added more html template and place holder file for change

### CODE REVIEW

File: scripts/deploy_gke.sh [hunk 1]
```diff
@@ -96,8 +96,8 @@ fi
 CLUSTER_NAME=$CLUSTER_PREFIX-$REGION
 
 # Deploy to GKE
-kustomize edit set image gcr.io/$IMAGE_PROJECT/datacommons-website=gcr.io/$IMAGE_PROJECT/datacommons-website:$WEBSITE_HASH
-kustomize edit set image gcr.io/$IMAGE_PROJECT/datacommons-nl=gcr.io/$IMAGE_PROJECT/datacommons-nl:$WEBSITE_HASH
+kustomize edit set image gcr.io/datcom-ci/datacommons-website=gcr.io/$IMAGE_PROJECT/datacommons-website:$WEBSITE_HASH
+kustomize edit set image gcr.io/datcom-ci/datacommons-nl=gcr.io/$IMAGE_PROJECT/datacommons-nl:$WEBSITE_HASH
```
Comment (beets) [inline scripts/deploy_gke.sh:100]: hm, so do we no longer need the IMAGE_PROJECT var set above?

**Review by beets — APPROVED** [review]
could you add more description to the PR around how this fixes autopush? thanks!

File: scripts/deploy_gke.sh (hunk 1 above)
Comment (shifucun) [reply scripts/deploy_gke.sh:100]: It's still used, but only in the replaced string. `kustomize edit set image <default> <new>` where the default on is always datcom-ci.

**Comment by shifucun** [comment]
Thanks for review!

**Pull request merged**